packages/core/src/engines/index.ts

```diff
@@ -417,10 +418,11 @@ export const installEngines = async (failOnMissingDeps?: boolean): Promise<boole
     const enginesToInstall: Array<RnvEngineInstallConfig> = [];
     const readyEngines: Array<string> = [];
     const engineConfigs: Array<RnvEngineInstallConfig> = [];
-    // if (filteredEngines) {
 
     Object.keys(filteredEngines).forEach((k) => {
-        const engineRootPath = doResolve(k);
+        const pathLookups = generateLookupPaths(k);
```

Comment:
@ElenaDiachenko can you add description why was this change needed for future reference